core/block_svg.ts

```diff
@@ -1012,8 +1012,8 @@ export class BlockSvg
     if (type.equals(MutatorIcon.TYPE)) this.mutator = null;
 
     if (this.rendered) {
-      // TODO: Change this based on #7068.
-      this.render();
+      this.queueRender();
+      renderManagement.triggerQueuedRenders();
```

Comment:
You can move the trigger into `queueRender` since you're always doing them together.

Reply:
I take it back--in a later PR you aren't doing them together.

Reply:
In this PR yes! But there are other places throughout the `BlockSvg` class where I don't trigger an immediate render. I.e. I use the render management system properly :P